.env.sample

```diff
@@ -85,7 +78,7 @@ STORAGE_BACKEND=s3
 #######################################
 # S3 Backend
 #######################################
-STORAGE_S3_BUCKET=name-of-your-s3-bucket
+STORAGE_S3_BUCKET=supa-storage-bucket
```

Comment:
default value for bucket name which is the same as the bucket we create on the docker-compose.
This makes it easy to simply rename the `.env.sample` to `.env` without any additional changes